leshan-core-cf/src/main/java/org/eclipse/leshan/core/californium/EndpointFactory.java

```diff
@@ -28,6 +30,8 @@
 public interface EndpointFactory {
 
     CoapEndpoint createUnsecuredEndpoint(InetSocketAddress address, NetworkConfig coapConfig, ObservationStore store);
+    
+    CoapEndpoint createOSCoreEndpoint(InetSocketAddress address, NetworkConfig coapConfig, ObservationStore store, HashMapCtxDB db);
```

Comment:
for now my feeling is that there no need to add createOSCoreEndpoint maybe just add the db parameter to both other methods (but as the californium API is not yet finished, I maybe missed something)

Reply:
Yes I removed this new method and added the db parameter to the others instead. Done in commit 495d5019f750ffee544ce7a66bb27c270e372de7